kivy_ios.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,138 @@
+.gitignore
+CHANGELOG.md
+LICENSE
+MANIFEST.in
+README.md
```

Comment:
do we really want the `kivy_ios.egg-info/` directory to be tracked or this was committed by mistake?

Reply:
Ooops, mistake :)